certora/specs/MintBurnEthereum.spec

```
@@ -0,0 +1,135 @@
+import "Delegation.spec";
+
+/*
+┌─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
+│ Rules: only the token holder or an approved third party can reduce an account's balance                             │
+└─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
+*/
+rule onlyAuthorizedCanTransfer(env e, method f) filtered {
```

```suggestion
rule onlyAuthorizedCanTransfer(env e, method f, calldataarg args) filtered {
```
and remove the calldataarg definition below. The convention for now is: if a rule is about a function, then add all arguments of this function to the parameters of the rule

---

For the record, no doing this change as we agreed to preserve simpler diffs w.r.t to OZ's original spec.